apps/hestia/src/components/trading/Orders/index.tsx

```diff
@@ -47,6 +47,12 @@ export const Orders = () => {
   } = useProfile();
   const connected = tradeAddress?.length > 0;
 
+  const isFundingButton = useMemo(() => {
+    if (!mainAddress) return true;
+    if (!tradeAddress) return false;
+    return true;
+  }, [mainAddress, tradeAddress]);
```

Comment:
Is there any need to check `if (!mainAddress) return true;` first?

Can it be simplified to:
```ts
const isFundingButton = useMemo(() => {
    return !!tradeAddress;
}, [tradeAddress]);
```

Reply:
Well....Yes actually. One dependency could be removed here. I rather choose to removed `tradeAddress` dependency here. 

```js
const isFundingType = useMemo(() => !mainAddress, [mainAddress]);
```